README.md

```diff
@@ -1,26 +1,63 @@
-# Nimiq Go Client
+# Nimiq Go RPC Client
+[![Report card](https://goreportcard.com/badge/github.com/nimiq-community/go-client)](https://goreportcard.com/report/github.com/nimiq-community/go-client)
+[![GoDoc](https://godoc.org/github.com/nimiq-community/go-client?status.svg)](https://godoc.org/github.com/nimiq-community/go-client)
+
 
 > Go implementation of the Nimiq RPC client specs.
 
-## Usage
+## About
+A Nimiq RPC client library in Go. This client library implements the [Nimiq RPC specification](https://github.com/nimiq/core-js/wiki/JSON-RPC-API#remotejs-client). This client uses the jsonrpc library to handle JSON-RPC 2.0 requests and responses. For more information about this library see the [jsonrpc documentation](https://godoc.org/github.com/ybbus/jsonrpc)
+
+### What is Nimiq?
+A blockchain technology inspired by Bitcoin but designed to run in your browser. It is money by nature but capable to do much more.
+See [the Nimiq website](https://nimiq.com) for more information.
+
+### Usage
+This library is fully Go module compatible. See the full documentation on [GoDoc](https://godoc.org/github.com/nimiq-community/go-Client)
+An example of using this library can be found below:
+
+```
+package main
```

Comment:
For the sake of simplicity, let's just do three lines for the example

```go
client := nimiqrpc.NewClient("address.to.nimiqnode.com")
balance, _ := client.GetBalance("NQ52 V4BF 52J3 0PM6 BG4M 9QY1 RUYS UAL6 CJD2")
fmt.Println("Balance: ", balance)
```

Reply:
I can live with that